pkgs/applications/networking/browsers/mullvad-browser/default.nix

```diff
@@ -42,6 +42,17 @@
 , mediaSupport ? true
 , ffmpeg
 
+# Updater
+, writeShellScript
+, coreutils
+, curl
+, gnugrep
+, gnused
+, gnupg
+, nix
+, common-updater-scripts
```

Comment:
Maybe I overlook something, but now that you use `callPackage`, are these still needed?

```suggestion
```